cardsWorthTen.js

```diff
@@ -21,7 +72,25 @@ const cards = [
  * @param {array} cards
  * @return {string} displayVal
  */
-const cardsWorthTen = cards => {};
+const cardsWorthTen = cards => {
+
+  let filteredCards = cards.filter((card) => card.val === 10);
+  let filteredVal = [];
+
```

Comment:
also a hint on how you might filter values and make a string . . . You have the array of card objects already. In your loop below you are reaching the value correctly. Now instead of console logging them, tack them on to the end of a new collection. No single way to do this, you could start with an empty string and concatenate on to the end . .  or perhaps make an empty array that you push the new values into, and then later use `Array.join(', ')` to convert the array into a string of those values separated by a comma and a space.